decentraland/helpers/index.js

```diff
@@ -152,6 +152,29 @@ export const formatNumber = number => {
   return formattedNumber;
 };
 
+export const removeFromList = (list, toRemove) =>
+  list.filter(e => e !== toRemove);
+
+export const listsAreEqual = (first, second) => {
+  if (first.length !== second.length) return false;
+
+  return (
+    first.every(e => second.includes(e)) && second.every(e => first.includes(e))
+  );
+};
+
+export const openURL = async url => {
+  const supported = await Linking.canOpenURL(url);
```

Comment:
Is this the one that shows a modal of a browser over the app or kicks the user out to their default browser? I know both options are available to us with different libs

Reply:
It kicks the user to the browser (as a user I like that option more), which should be the desirable behavior?

Reply:
I agree, I like that option more too as a user.

Reply:
Feel free to resolve this.

Reply:
Resolving this.